src/main/java/edu/dbsleipzig/stream/grouping/application/CitiBikeExample.java

```diff
@@ -56,14 +63,14 @@ public static void main(String[] args) throws Exception {
 
         // Configure and build the grouping operator
         GraphStreamGrouping groupingOperator = new TableGroupingBase.GroupingBuilder()
-          .setWindowSize(15, WindowConfig.TimeUnit.DAYS)
-          .addVertexGroupingKey(":label")
-          .addEdgeGroupingKey("gender")
-          .addEdgeGroupingKey(":label")
-          .addVertexAggregateFunction(new Count())
-          .addEdgeAggregateFunction(new Count())
-          .addEdgeAggregateFunction(new AvgProperty("tripduration"))
-          .build();
+                .setWindowSize(15, WindowConfig.TimeUnit.DAYS)
```

Comment:
Why indentation changes? Leave as it is. Default indentations should be 2 spaces